src/actions/resize/IResizeAction.ts

```diff
@@ -4,4 +4,6 @@ import {GravityParam} from "../../params/gravity/Gravity";
 export interface IResizeAction extends IAction{
   aspectRatio(ratio:string | number):IResizeAction;
   gravity(type:GravityParam):IResizeAction;
+  height(x:number):IResizeAction;
```

Comment:
Can't this also be a string?

Reply:
Yea I think it can actually, but height and width. and the constructor values as well